packages/components/Text.tsx

```diff
@@ -0,0 +1,13 @@
+import type { ReactNode } from "react";
+
+type SupportedTags = "p" | "span";
+
+interface Props {
+  children?: ReactNode;
+  as?: SupportedTags;
+  className?: string;
+}
+
+export function Text({ children, as: Tag = "p", className = "" }: Readonly<Props>) {
+  return <Tag className={className}>{children}</Tag>;
+}
```

Comment:
Are we merging the code with the POC?

Reply:
Using tailwind directly on a p tag in the component CallToAction seems to make more sense to me

Reply:
Made in: https://github.com/podcodar/webapp/pull/173/commits/6e09480243d054427b80bc346523a31c784e6a01

Reply:
Although I understood, I kept wondering if we were in a new standard. I learned a lot here about all the code.